service/src/users-api.js

```diff
@@ -99,9 +106,10 @@ exports.register = (app) => {
   app.put("/api/users/:id", checkJwt, middleware.Authz(), async (req, res) => {
     const user = req.body;
     const id = req.params.id;
-    const response = await directory.updateUser(req, id, user);
+    const response = await directory.updateUser(req, id,  user);
     if (response) {
-      res.status(201).send(response);
+      const data = toJson(SetObjectResponseSchema, response).result
```

Comment:
same here, response should be sendable by this route handler (all of the complexity that is necessary should be encapsulated by `directory.updateUser`)